### PR TITLE
refactor: a bunch of trivial-ish code improvements

### PR DESCRIPTION
**Pull request by UdjinM6**
## Checklist:
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [ ] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: src/governance/vote.h [hunk 1]
```diff
@@ -63,27 +63,21 @@ class CGovernanceVote
...
 private:
-    bool fValid;     //if the vote is currently valid / counted
-    bool fSynced;    //if we've sent this to our peers
-    int nVoteSignal; // see VOTE_ACTIONS above
+    int nVoteSignal{VOTE_SIGNAL_NONE}; // see VOTE_ACTIONS above
     COutPoint masternodeOutpoint;
     uint256 nParentHash;
-    int nVoteOutcome; // see VOTE_OUTCOMES above
-    int64_t nTime;
+    int nVoteOutcome{VOTE_OUTCOME_NONE}; // see VOTE_OUTCOMES above
```
Comment (knst) [inline src/governance/vote.h:69]: to update type here also?

```
-int nVoteSignal{VOTE_SIGNAL_NONE}; // see VOTE_ACTIONS above
+vote_signal_enum_t  nVoteSignal{VOTE_SIGNAL_NONE}; // see VOTE_ACTIONS above
...
-int nVoteOutcome{VOTE_OUTCOME_NONE}; // see VOTE_OUTCOMES above
+vote_outcome_enum_t nVoteOutcome{VOTE_OUTCOME_NONE}; // see VOTE_OUTCOMES above
```
?

**Review by knst — APPROVED** [review]
LGTM efc8c99139659695ebfff2139645f92b8055b2f4

File: src/governance/vote.h [hunk 2]
```diff
@@ -18,24 +18,25 @@ class CKeyID;
 class PeerManager;
 
 // INTENTION OF MASTERNODES REGARDING ITEM
-enum vote_outcome_enum_t : uint8_t {
-    VOTE_OUTCOME_NONE      = 0,
-    VOTE_OUTCOME_YES       = 1,
-    VOTE_OUTCOME_NO        = 2,
-    VOTE_OUTCOME_ABSTAIN   = 3
+enum vote_outcome_enum_t : int {
```
Comment (knst) [inline src/governance/vote.h:21]: From [N4659 C++ 7.2/5](https://timsong-cpp.github.io/cppwp/n4659/dcl.enum#7):

>    For an enumeration whose underlying type is not fixed, the underlying type is an integral type that can represent all the enumerator values defined in the enumeration. If no integral type can represent all the enumerator values, the enumeration is ill-formed. It is implementation-defined which integral type is used as the underlying type except that the underlying type shall not be larger than int unless the value of an enumerator cannot fit in an int or unsigned int. If the enumerator-list is empty, the underlying type is as if the enumeration had a single enumerator with value 0.

nit: type can be omitted here completely:
```
-enum vote_outcome_enum_t : int {
+enum vote_outcome_enum_t {
```

**Review by knst — APPROVED** [review]
utACK c32fac079f8b887b9a6e274a2eb3b47c5ee01d93

**Review by PastaPastaPasta — APPROVED** [review]
utACK c32fac079f8b887b9a6e274a2eb3b47c5ee01d93

**Pull request merged**